File: sdkwork-ai-sdk-go/types/knowledge_base_response.go
package types

// 知识库响应
type KnowledgeBaseResponse struct {
	Name string `json:"name"`
	Description string `json:"description"`
	Status string `json:"status"`
	KnowledgeBaseId string `json:"knowledge_base_id"`
	EmbeddingModelId string `json:"embedding_model_id"`
	IndexType string `json:"index_type"`
	CreatedAt int `json:"created_at"`
	UpdatedAt int `json:"updated_at"`
	DocCount int `json:"doc_count"`
}

File: sdkwork-ai-sdk-csharp/Models/KnowledgeBaseResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class KnowledgeBaseResponse
    {
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? Status { get; set; }
        public string? KnowledgeBaseId { get; set; }
        public string? EmbeddingModelId { get; set; }
        public string? IndexType { get; set; }
        public int? CreatedAt { get; set; }
        public int? UpdatedAt { get; set; }
        public int? DocCount { get; set; }
    }
}
